policies/templates/gcp_iam_audit_log.yaml

```diff
@@ -0,0 +1,98 @@
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# GCPIAMAuditLogConstraintV1 verifies that audig logging is enabled in the
+# given service with the given log types.
+
+apiVersion: templates.gatekeeper.sh/v1alpha1
+kind: ConstraintTemplate
+metadata:
+  name: gcp-iam-audit-log-v1
+spec:
+  crd:
+    spec:
+      names:
+        kind: GCPIAMAuditLogConstraintV1
+        plural: gcpiamauditlogconstraintsv1
+      validation:
+        openAPIV3Schema:
+          properties:
+            service:
```

Comment:
if user wants to make sure DATA_READ & DATA_WRITE is enabled on say 5 apis, do we expect the user to create 5 constraints for that or should we allow them to specify that in one constraint? (i.e. having service as a list instead)

Reply:
Done. We now accept a services list.